tests/robustness/coverage/README.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+Go tests in this directory analyze the usage of Etcd API based on collected
+traces from a Kubernetes cluster. They output information on:
+
+1. Number of calls per gRPC method used by Kubernetes
+
+This information can be used to track the coverage of k8s-etcd contract.
+
+### Manual test execution
+
+At first we will manually set up the cluster, run e2e tests, download traces and
+then execute the test.
+
+1\. Set up [KIND
+cluster](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) with
+tracing exporting to [Jaeger](https://www.jaegertracing.io/)
+
+```
+export KUBECONFIG="$(pwd)/kind-with-tracing-config"
+kind create cluster --config kind-with-tracing.yaml
+kubectl run jaeger --overrides='{ "apiVersion": "v1", "spec": { "hostNetwork": true, "nodeName": "kind-control-plane", "tolerations": [{"effect": "NoExecute", "operator": "Exists"}]} }' --labels='tier=control-plane' --image jaegertracing/jaeger:2.6.0
+```
+
+2\. Build and run Conformance tests from Kubernetes repository (this usually
+takes 30-40m or will time out after 1 hour). It needs to be run from Kubernetes git repository:
```

Comment:
We should add note that `Conformance tests` could be replaced with user prefered way to exercise K8s API. Conformance tests were just selected because ... (provide reasons).

Reply:
Updated to state that the goal is exercising Kubernetes API surface. Conformance tests are used, because they are a good starting point to achieve this, so I marked them as an example

Reply:
What I'm saying we should document that. In the step.